src/Compilers/VisualBasic/Portable/Symbols/Source/LocalSymbol.vb

```diff
@@ -404,6 +404,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Private ReadOnly Property ILocalSymbol_IsForEach As Boolean Implements ILocalSymbol.IsForEach
```

Comment:
>ILocalSymbol_IsForEach [](http://example.com/codeflow?start=34&length=22)

Please add tests for the VB implementations, similar to the C# tests.

Reply:
8264dc471d3eed9942e91da377ab0f166196a42e